web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -51,13 +53,28 @@ def resolve_available_office_hours(self, info, **kwargs):
         user = info.context.user
         filter_kwargs = {
             'finalist__isnull': True,
-            'date__gte': timezone.now(),
-            'start_time__gte': timezone.now().time()
         }
+        now = timezone.now()
```

Comment:
something I just noticed - a user who sees their own profile page should also see available mentor office hours (currently it isn't the case). and also superusers, probably.